README.md

```diff
@@ -171,7 +171,7 @@ Navigate to the root directory of the jpo-ode-private project:
 ```
 It is important you run `mvn clean` first and _then_ `mvn install` because clean installs the required OSS jar file in your local maven repository.
 
-**(Optional)**: Familiarize yourself with Docker and follow the instructions in the [docker/README.me].
```

Comment:
I went ahead and did this in branch ode-381. No need to be implemented in any other branch.